FHEM/lib/SD_ProtocolData.pm

```diff
@@ -858,6 +858,7 @@ package lib::SD_ProtocolData;
 				length_max		=> '12',				# message has only 10 bit but is paddet to 12
 			},
 		"31"	=>	## Pollin ISOTRONIC - 12 Tasten remote
+							# ! Protocol is very similar to ID 34 !
```

Comment:
Warum nicht gleich auskommentieren?

Reply:
Weil ich auf Basis

```
# u31#891EE   MU;P0=-9584;P1=592;P2=-665;P3=1223;P4=-1311;D=01234141412341412341414123232323412323234;CP=1;R=0;
# u31#891FE   MU;P0=-12724;P1=597;P2=-667;P3=1253;P4=-1331;D=01234141412341412341414123232323232323232;CP=1;R=0;
```
und zusätzlich https://github.com/RFD-FHEM/RFFHEM/issues/44#issuecomment-444258173

testete ob diese bei ID 34 eintreffen, was nicht der Fall war.
Bin unsicher ob es ebven das gleiche wie 34 ist, aber wir es nicht wiederlegen können, der Schritt zu => y. Wir wissen ja nicht, ob doch irgendwo in den Weiten es nutzt ....

Reply:
Doch, doch, die passen schon, wenn du bei P34 noch
	reconstructBit  => '1',
dazu nimmst. Das hatten wir damals noch nicht.

Reply:
Davon war noch keine Rede ;-) Mit diesen Informationen sieht es natürlich ganz anders aus.

Reply:
Nicht aufgepasst :-)
https://github.com/RFD-FHEM/RFFHEM/issues/716#issuecomment-565720148

Nimm bitte clockabs gleich noch auf 635 - das passt besser.